### DISCUSSION TITLE
Error while setting up the project

### DISCUSSION THREAD

**Discussion by harshalvk** [openstatusHQ/openstatus#931] [General]
![image](https://github.com/user-attachments/assets/dfe89c71-e230-4490-affd-5d89539ae3b1)
I am facing the issue while running the project, could not resolve the problem

**Comment by mxkaske** [comment]
Hey @Harshalvk, from the error code, it looks like you are missing the database url.

**Reply by harshalvk** [reply]
Hello @mxkaske , I have added the database url but still facing the issue
![image](https://github.com/user-attachments/assets/94a87110-ddc3-4937-ad7c-99abc73369dc)

**Reply by MdTanwer** [reply]
hi sir    have you set up  in local mashine sucessfully

**Reply by thibaultleouay** [reply]
You should use https url instead of libsql

**Reply by MdTanwer** [reply]
thanks

**Comment by MdTanwer** [comment]
![image](https://github.com/user-attachments/assets/f8e59b10-774e-4ad9-850a-0f9f86f2916c)
i face this issue pls help me when i try to setup in locally

**Comment by MdTanwer** [comment]
readme.md  is very confusing

**Reply by thibaultleouay** [reply]
Sorry @MdTanwer 

Let me know how we can help to improve the readme 

Maybe our docs are a bit better ?

**Reply by MdTanwer** [reply]
To improve the README,  I think you should clearly mention that the project should be run in Windows Subsystem for Linux (WSL). This will help users who are working on Windows to set up and run the project more smoothly. This way, our documentation will be more user-friendly and comprehensive.
